database/migrations/008_optimistic.go

```diff
@@ -0,0 +1,53 @@
+package migrations
+
+import (
+	"github.com/flashbots/mev-boost-relay/database/vars"
+	migrate "github.com/rubenv/sql-migrate"
+)
+
+var Migration008Optimistic = &migrate.Migration{
+	Id: "008-optimistic",
+	Up: []string{
+		`
+		ALTER TABLE ` + vars.TableBuilderBlockSubmission + ` ADD optimistic_submission bool NOT NULL default false;
+		ALTER TABLE ` + vars.TableBuilderBlockSubmission + ` ADD decode_duration       bigint NOT NULL default 0;
+		ALTER TABLE ` + vars.TableBuilderBlockSubmission + ` ADD prechecks_duration    bigint NOT NULL default 0;
+		ALTER TABLE ` + vars.TableBuilderBlockSubmission + ` ADD simulation_duration   bigint NOT NULL default 0;
+		ALTER TABLE ` + vars.TableBuilderBlockSubmission + ` ADD redis_update_duration bigint NOT NULL default 0;
+		ALTER TABLE ` + vars.TableBuilderBlockSubmission + ` ADD total_duration        bigint NOT NULL default 0;
+	`, `
+		ALTER TABLE ` + vars.TableBlockBuilder + ` ADD is_optimistic  bool NOT NULL default false;
```

Comment:
maybe index on `is_optimistic` too?

Reply:
again i think we are ok to not! block builder table is super small, so no big queries on this.